lib/sales_engine.rb

```diff
@@ -1,19 +1,32 @@
 require 'csv'
 require_relative 'merchant_repository'
 require_relative 'merchant_parser'
+require_relative 'item_repository'
+require_relative 'item_parser'
 
 class SalesEngine
+  DEFUALT_DATA_LOCATIONS = {
+                              merchant: "./data/merchants.csv",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.